content-repo/extra-docs/packs/palo-alto-networks-cortex-xdr---investigation-and-response.md

```diff
@@ -1,32 +1,43 @@
+The Cortex XDR by Palo Alto Networks pack automates Cortex XDR incident response, and includes custom Cortex XDR incident views and layouts to aid analyst investigations.
 
-Cortex XDR is a detection and response app that natively integrates network, endpoint, and cloud data to stop sophisticated attacks. 
+Cortex XDR is a detection and response app that natively integrates network, endpoint, and cloud data to stop sophisticated attacks.
 
-Responding and managing these attacks requires security teams to reconcile data from multiple sources. Valuable time is lost shuttling between screens and executing repeatable tasks while an attack continues to manifest. 
+Effectively responding to these attacks requires security teams to consolidate data from multiple sources. Critical time is wasted switching between tools and performing repetitive tasks, allowing the attack to escalate further.
+
+This pack can provide significant assistance.
+
+## What does the pack do?
+
+- **Automated Incident Synchronization**: Keeps incidents consistently updated between Cortex XDR and Cortex XSOAR, maintaining data integrity and reducing manual effort.
+- **Comprehensive Alert Management**: Automates alert handling, including enrichment, severity assessment, and remediation, for a streamlined incident response process.
+- **Advanced Threat Hunting**: Leverages the XQL Query Engine for in-depth data analysis, enabling proactive threat hunting and investigation.
+- **Ready-to-Use Playbooks**: a range of pre-built playbooks optimized for common security workflows, allowing teams to quickly deploy effective responses with minimal configuration.
+
+The playbooks included in this pack can help you save time and keep your incidents and indicators in sync with Cortex XDR. They also help automate repetitive tasks:
 
-The playbooks included in this pack help you save time and keep your incidents in sync with Cortex XDR. They also help automate repetitive tasks associated with Cortex XDR incidents:
 - Syncs and updates Cortex XDR incidents.
 - Triggers a sub-playbook to handle each alert by type.
 - Extracts and enriches all relevant indicators from the source alert.
 - Hunts for related IOCs.
-- Calculates the severity of the incident.
-- Interacts with the analyst to choose a remediation path or close the incident as a false positive based on the gathered information and incident severity.
-- Remediates the incident by blocking malicious indicators and isolating infected endpoints.
+- Assesses the severity of the incident.
+- Collaborates with the analyst to determine the appropriate remediation path or classify the incident as a false positive, based on gathered data and incident severity.
+- Remediates the incident by blocking malicious indicators and isolating compromised endpoints.
+
+## The Palo Alto Networks Cortex XDR - Investigation and Response pack enables the following flows:
```

Comment:
What is the correct name of the pack? 
Is it The Cortex XDR by Palo Alto Networks pack  or The Palo Alto Networks Cortex XDR - Investigation and Response pack?  Make sure the correct name is used everywhere.